docs/modules/dids.md

```diff
@@ -47,7 +47,7 @@ const { jws, linkedBlock } = await did.createDagJWS(payload)
 const jwsCid = await ipfs.dag.put(jws, { storeCodec: 'dag-jose', hashAlg: 'sha2-256' })
 
 // put the payload into the ipfs dag
-const block = await ipfs.block.put(linkedBlock, { cid: jws.link })
+const block = await ipfs.block.put(linkedBlock, { format: 'dag-cbor' })
```

Comment:
Can you put a block with format? I thought blocks needed a CID?

Reply:
not sure at what point it changed - https://github.com/ipfs/js-ipfs/blob/master/docs/core-api/BLOCK.md#parameters-1

assuming it just uses codec/hash to derive cid it self instead now

Reply:
ok 👍